cmd/stack_create.go

```diff
@@ -65,7 +65,7 @@ The stack name must start with a lowercase letter, and can contain only lowercas
 
 			stackPath := filepath.Join(config.ProjectDir, stack)
 
-			match, err := IsValidProjectName(stack)
+			match, err := IsValidParamName(stack, "project-name")
 			if !match {
 				return err
```

Comment:
Could we not wrap this error here with info on the `project-name`, instead of passing it everytime?

Reply:
Like we do in https://github.com/appsody/appsody/pull/921/files#diff-4d78da15d8ee41e84a70fdd87c0481e5R61